# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1471361696 -7200
#      Tue Aug 16 17:34:56 2016 +0200
# Node ID b5ae7d55429c7e40491ab20ac21a9a3d61bf849f
# Parent  46d39db0c57101a39ad72068f200b493d53e5ab3
dump_demos.m: Replace rows with size ( ,1) for Matlab compatibility

diff --git a/scripts/testfun/private/dump_demos.m b/scripts/testfun/private/dump_demos.m
--- a/scripts/testfun/private/dump_demos.m
+++ b/scripts/testfun/private/dump_demos.m
@@ -263,19 +263,19 @@ function dump_helper_fcns (fid)
 "  if (nargin == 1)                                                           "
 "    style = 'profile';                                                       "
 "  end                                                                        "
 "                                                                             "
 "  switch (lower (style))                                                     "
 "    case 'profile'                                                           "
 "      htmp = plot (cmap(:,1),'r', cmap(:,2),'g', cmap(:,3),'b');             "
 "      set (gca (), 'ytick', 0:0.1:1);                                        "
-"      set (gca (), 'xlim', [0 rows(cmap)]);                                  "
+"      set (gca (), 'xlim', [0 size(cmap,1)]);                                "
 "    case 'composite'                                                         "
-"      htmp = image (1:rows(cmap));                                           "
+"      htmp = image (1:size(cmap,1));                                         "
 "      set (gca, 'ytick', []);                                                "
 "      colormap (cmap);                                                       "
 "  end                                                                        "
 "  xlabel ('color index');                                                    "
 "                                                                             "
 "  if (nargout > 0)                                                           "
 "    h = htmp;                                                                "
 "  end                                                                        "
